pkg/runner/cypress.go

```diff
@@ -73,6 +74,12 @@ func (r *CypressRunner) Run(execution testkube.Execution) (result testkube.Execu
 		return result, err
 	}
 
+	for key, value := range execution.Params {
+		if err = os.Setenv(key, value); err != nil {
```

Comment:
hm... perhaps https://docs.cypress.io/guides/guides/environment-variables#Option-4-env is a better way to pass these!?

Reply:
ok, let's do that